modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -12,12 +11,8 @@ ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
 :ash:
 :cp: Azure Stack Hub
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-user-provisioned"]
-:azure:
-:cp: Azure
-endif::[]
```

Comment:
This module is still used by `installing-restricted-networks-azure-user-provisioned`. Did you remove this by mistake?